src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -96,6 +96,20 @@ const ShowPracticeQuestions = () => {
     const currentSectionId = section ? section.id : page ? page.id : null;
     return currentSectionId ? getNextPageWithPracticeQuestions(currentSectionId, locationFilters, book) : undefined;
   }, [book, page, section, locationFilters]);
+  const questionsInProggress = useSelector(pqSelectors.questionsInProggress);
+  const hasAnswers = useSelector(pqSelectors.hasAnswers);
+
+  const practiceQuestionsContent = (
+    <QuestionsWrapper>
+      <QuestionsHeader>
+        <FormattedMessage id='i18n:practice-questions:popup:questions'>
+          {(msg: string) => msg}
+        </FormattedMessage>
+      </QuestionsHeader>
+      <ProgressBar total={questionsCount} activeIndex={currentQuestionIndex} />
+      {questionsInProggress ? <Question /> : <IntroScreen />}
+    </QuestionsWrapper>
+  );
```

Comment:
now you can move this component into the render

Reply:
What exactly do you mean?

Reply:
This shouldn't be a const anymore
Since now it is used only in one place you can replace `practiceQuestionsContent` with the jsx